### PR TITLE
Re-Structure & Add SASS

### PR DESCRIPTION
**Pull request by thanhtranit**

### CODE REVIEW

File: components/breadcrumb/breadcrumb.component.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
+(function (ngxBootstrap, fileService) {
+  ngxBootstrap.ngxComponents.ngxBreadcrumbComponent = ng.core.Directive({
+    selector: 'ngx-breadcrumb'
+  })
+  .Class(breadcrumb());
+
+  function breadcrumb() {
+    return {
+      constructor: [ng.core.ElementRef, function (elementRef) {
+        this.nativeElement = elementRef.nativeElement;
```
Comment (Leon-Kennedy) [inline components/breadcrumb/breadcrumb.component.js]: Shouldn't assign anything to this. It'll bring issues to you when you do inherit it
I'm updating README.md, there'll rules we need to follow to code inheritable components

I'm ping you on Gitter when I finish updating README.md

File: index.local.html [hunk 2]
```diff
@@ -1,4 +1,4 @@
-﻿<!doctype html>
+<!doctype html>
```
Comment (Leon-Kennedy) [inline index.local.html]: Why did you rename this file to index.local.html ?

File: ngx-bootstrap.js [hunk 3]
```diff
@@ -14,6 +14,11 @@
 
       return this.COMPONENTS + componentName + '/tests/ui/' + testcase + '/test-case.html';
     };
+    this.getComponentStyle = function (componentName, styleName) {
```
Comment (Leon-Kennedy) [inline ngx-bootstrap.js]: Could you move util functions to a separated file ? 
Notice that we should add all util functions to global variable ngxBootstrap

File: index.local.html (hunk 2 above)
Comment (thanhtranit) [inline index.local.html]: We will avoid conflicts when multiple people commit index.html.
If you develop, please copy and rename this file to index.html and don't commit index.html

File: ngx-bootstrap.js (hunk 3 above)
Comment (thanhtranit) [inline ngx-bootstrap.js]: OK. I will.

File: index.local.html (hunk 2 above)
Comment (Leon-Kennedy) [inline index.local.html]: I have added index.html to .gitgone file. It mean that there's nobody can commit this file. 
Could you check again if it's working as I've mentioned ?

Comment (thanhtranit) [inline index.local.html]: You must remove this file and commit it to take effect.
If you do so, others will not see your template and we need one more file "index.local.html"

If you don't remove and commit this file, you can change some things and push to test right now

File: ngx-bootstrap.js (hunk 3 above)
Comment (thanhtranit) [inline ngx-bootstrap.js]: I have moved util functions to a separated file.
I have also moved ngx-bootstrap.js and ngx-bootstrap.utils to components folder.

File: components/breadcrumb/breadcrumb.component.js (hunk 1 above)
Comment (thanhtranit) [inline components/breadcrumb/breadcrumb.component.js]: I'm waiting for you!

File: index.local.html (hunk 2 above)
Comment (Leon-Kennedy) [inline index.local.html]: Hmmm....Ok, let me check my gulp task again

Comment (thanhtranit) [inline index.local.html]: You can ask @huuthonguyen76 to testing

File: components/breadcrumb/breadcrumb.component.js (hunk 1 above)
Comment (Leon-Kennedy) [inline components/breadcrumb/breadcrumb.component.js]: I've already pushed a commit to develop branch. Please read README.md

File: index.local.html (hunk 2 above)
Comment (Leon-Kennedy) [inline index.local.html]: I've already push a commit to develop branch to fix this gulp task. Please refresh your branch from develop
